packages/swapper/src/swappers/PortalsSwapper/utils/helpers.ts

```diff
@@ -1,8 +1,84 @@
-import type { ChainId } from '@shapeshiftoss/caip'
+import { type AssetId, type ChainId, foxAssetId, toAccountId } from '@shapeshiftoss/caip'
+import type { EvmChainId } from '@shapeshiftoss/chain-adapters'
+import { KnownChainIds } from '@shapeshiftoss/types'
 
 import type { PortalsSupportedChainId } from '../types'
 import { PortalsSupportedChainIds } from '../types'
 
+const WELL_FUNDED_ADDRESS = '0x267586F48043e159624c4FE24300c8ad2f352fc7'
```

Comment:
Who has the control over this address? Having it here in a constant seems magic but it might be how we usually do

Reply:
I do - it's an account 0+ (like... very much plus i.e 10 or so which means it won't automagically appear on native) of mine on my main testing wallet which I plan to remove in MM once this goes in)